docs/build.md

```diff
@@ -592,6 +593,44 @@ spec:
     timestamp: SourceTimestamp
 ```
 
+### Defining the vulnerabilityScan
+
+`vulnerabilityScan` provides configurations about running a scan for your generated image.
+- `vulnerabilityScan.enabled` - Specify whether to run vulnerability scan for image. The supported types are true and false.
+- `vulnerabilityScan.fail` - indicates whether to fail the build run if the vulnerability scan results in vulnerabilities. The supported types are true and false.
+- `vulnerabilityScan.ignore.issues` - references the security issues to be ignored in vulnerability scan
+- `vulnerabilityScan.ignore.severity` - indicates the severities of security issues to be ignored (comma separated). Supported types are : Low, Medium, High, Critical.
```

Comment:
This is not what I intended. In the ship, I wrote:

```yaml
  severity: Low | Medium | High | Critical
```

So, the value is either low or medium or high or critical, but not multiple of them. Then, we would chose an array. Severities have an order. If Low is specified, then only Low should be ignored. If Medium is defined, then Low and Medium are ignored. Etc.

That was my idea.

Side note: while just checking this in the SHIP, I noticed that I used Low | Medium | High | Critical in the spec, while I used lower case medium and high in the status. We should be consistent and I tend to suggest to use fully lower-case everywhere.

Reply:
@SaschaSchwarze0 Shall we allow Critical here ? because if someone specifies critical, then it will ignore all the vulnerabilities.